modules/basics/src/main/java/com/opengamma/basics/BuySell.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (C) 2015 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.basics;
+
+import org.joda.convert.FromString;
+import org.joda.convert.ToString;
+
+import com.google.common.base.CaseFormat;
+import com.opengamma.collect.ArgChecker;
+
+/**
+ * Flag indicating whether a trade is "buy" or "sell".
+ * <p>
+ * Species whether the financial instrument is buy-side or sell-side.
```

Comment:
whether the financial instrument is buy-side or sell-side isn't particularly clear as to what it means in this context